### DISCUSSION TITLE
Auto-fix configured import aliases

### DISCUSSION THREAD

**Discussion by chbndrhnns** [astral-sh/ruff#6277] [Ideas]
There are a couple of import aliases I use quite often, like these:

```
import typing as t
import pydantic as p
import fastapi as f
```

I'd like to suggest that ruff can be configured to auto-fix my configured aliases.

**Comment by charliermarsh** [comment]
I believe you can customize these today (https://beta.ruff.rs/docs/settings/#flake8-import-conventions-aliases), with autofix via the https://beta.ruff.rs/docs/rules/unconventional-import-alias/ rule.

**Reply by chbndrhnns** [reply]
Wow, I definitely need to dig flake stuff, that's new to me. Thanks for the hint!
